docs/guide/custom_policy.rst

```diff
@@ -5,7 +5,33 @@ Custom Policy Network
 
 Stable baselines provides default policy networks (see :ref:`Policies <policies>` ) for images (CNNPolicies)
 and other type of input features (MlpPolicies).
-However, you can also easily define a custom architecture for the policy (or value) network:
+
+One way of customising the policy network architecture is to pass argument using ``policy_kwargs`` parameter:
+
+.. code-block:: python
+
+  import gym
+  import tensorflow as tf
+
+  from stable_baselines import PPO2
+
+  # Custom MLP policy of two layers of size 32 each with tanh activation function
+  policy_kwargs = dict(act_fun=tf.nn.tanh, net_arch=[32, 32])
+  # Create the agent
+  model = PPO2("MlpPolicy", "CartPole-v1", policy_kwargs=policy_kwargs, verbose=1)
+  # Retrieve the environment
+  env = model.get_env()
+  # Train the agent
+  model.learn(total_timesteps=100000)
+  # Save the agent
+  model.save("ppo2-cartpole")
+
+  del model
+  # the policy_kwargs are automatically loaded
+  model = PPO2.load("ppo2-cartpole")
+
+
+You can also easily define a custom architecture for the policy (or value) network:
```

Comment:
Now that we have policy_kwargs, the following code block seems rather confusing. Why should I create my custom policy class that just changes the parameters to its superclass? To justify custom policy classes we would need a more complex example here.
My first idea was a LSTM network with a MLP stream that skips over the LSTM cells. But then it becomes all very hard to read for an example ...

Reply:
weirdly enough my previous answer to this comment seems to have disappeared... anyway:
Good point, this was also what I thought when I finished that section.
To answer you, I'll take my use case as an example.
When I'm testing RL algo with custom policy, I usually have only one or two policies (a "big" one and a "shallow" one). Having custom policy class instead of passing kwargs allows to name my policies, so it makes it clear in the log or in the terminal which policy is being used.
I don't think we should complexify the example, however adding a note on the equivalence of the two ways, and where you should use one rather than the other could be a good idea, what do you think ?

Reply:
I agree 100%